File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    let count = 0;
    for(let i=0; i<=t.length-p.length; ++i){
        const value = t.slice(i, p.length + i);
        if(+value <= +p)
            ++count;
    }
    return count;
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    pLen = len(p)
    num_p = int(p)
    for i in range(len(t)-pLen+1) : 
        split_num = int(t[i:i+pLen])
        if split_num <= num_p:
            answer+=1
    return answer